shared/src/business/useCases/docketEntry/addPaperFilingInteractor.ts

```diff
@@ -64,69 +65,50 @@ export const addPaperFilingInteractor = async (
     });
 
   if (!isCaseConsolidationFeatureOn) {
-    consolidatedGroupDocketNumbers = [docketNumber];
+    consolidatedGroupDocketNumbers = [subjectCaseDocketNumber];
   }
 
-  const baseMetadata = pick(documentMetadata, [
-    'filers',
-    'partyIrsPractitioner',
-    'practitioner',
-  ]);
-
-  const [docketEntryId, metadata, relationship] = [
-    primaryDocumentFileId,
-    documentMetadata,
-    DOCUMENT_RELATIONSHIPS.PRIMARY,
-  ];
-
-  const readyForService = metadata.isFileAttached && !isSavingForLater;
-
-  if (!docketEntryId) {
-    throw new Error('Did not receive a primaryDocumentFileId');
-  }
+  const isReadyForService =
+    documentMetadata.isFileAttached && !isSavingForLater;
 
   const docketRecordEditState =
-    metadata.isFileAttached === false ? documentMetadata : {};
+    documentMetadata.isFileAttached === false ? documentMetadata : {};
+
+  const user = await applicationContext
+    .getPersistenceGateway()
+    .getUserById({ applicationContext, userId: authorizedUser.userId });
 
   let caseEntities = [];
-  for (let docketNo of consolidatedGroupDocketNumbers) {
-    const aCase = await applicationContext
+  let filedByFromLeadCase;
+
+  for (const docketNumber of consolidatedGroupDocketNumbers) {
+    const rawCase = await applicationContext
```

Comment:
I love how this functionality is brought into a useCaseHelper for the other interactors that file and serve documents. Just curious if you think there's room for that here.

Reply:
We did spend a significant amount of time trying to reuse the useCaseHelper to be used from this file, however things were JUST different enough that it made more sense to not reuse it! We would have been adding a lot more conditionals/branching otherwise, and it got to a point where reusing it seemed to be defeating the purpose of the refactoring.